### DISCUSSION TITLE
400 eror in API even though file gets transcribed on the playground

### DISCUSSION THREAD

**Discussion by architrathi** [deepgram/community#912] [General help] [answered]
I'm intermittently getting 400 error for some mp3 files (starting yesterday). The response on the Speech to Text endpoint is also always just 'Error: "Bad Request: Invalid data received."'. Unable to debug - all these files work OK and are transcribed on the deepgram playground.

Can anyone help on this please.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by architrathi** [reply]
Request id: 4df77c97-9874-4322-b772-67db50cb9e55 , f0c7374b-d9b9-406a-a8e7-b83b84567d2e , 3ad054b2-7f64-4c36-943c-d4bb67132997

```
curl \
  -X POST \
  -H "Authorization: Token my_token_here" \
  -H "Content-Type: application/json" \
  -d '{"url":"https://generatedassets.s3.amazonaws.com/media/68d48585-dc05-43ed-904e-356652e45247_Begging Him.mp3"}' \       
  "https://api.deepgram.com/v1/listen?smart_format=true&punctuate=true&paragraphs=true&diarize=true&detect_language=true&model=nova-2"
{"err_code":"Bad Request","err_msg":"Bad Request: Invalid data received.","request_id":"4df77c97-9874-4322-b772-67db50cb9e55"}%   
```

**Reply by jpvajda** [reply, marked as answer]
@architrathi I checked our logs and here are some suggestions based on the errors: 

`4df77c97-9874-4322-b772-67db50cb9e55`

`Error: Raw binary data was sent to Deepgram without both the encoding and sample_rate set.`

`Suggestion:` try setting the encoding and sample_rate in the request.

`f0c7374b-d9b9-406a-a8e7-b83b84567d2e`

`Error: pcm_s24le audio is not fully supported`

`Suggestion:` Try converting to a 16-bit encoding: `ffmpeg -i <in>.wav -acodec pcm_s16le <out>.wav` For more info, see https://github.com/orgs/deepgram/discussions/483#discussioncomment-7793889

`3ad054b2-7f64-4c36-943c-d4bb67132997`

`Error: pcm_s24le audio is not fully supported`

Suggestion: Try converting to a 16-bit encoding: `ffmpeg -i <in>.wav -acodec pcm_s16le <out>.wav` For more info, see https://github.com/orgs/deepgram/discussions/483#discussioncomment-7793889
